### PR TITLE
Create Signature into flow to create account

### PR DESCRIPTION
**Pull request by vinicios-gomes**
## CREATE SIGNATURE INSIDE CREATE ACCOUNT FLOW

Add signature and/or initials creation to the account creation flow.

### ADD NEW FEATURE
To add this feature in your LibreSign version
In your browser, run in the console.
```js
OCA.LibreSign.setFeature('manage_signatures')
```
### ENABLE FEATURE

In your browser, run in the console.

```js
OCA.LibreSign.enableFeature('manage_signatures')
```

make sure you have the following lines in your oc_appconfig data table.

| appid     | configkey        | configvalue |
|-----------|------------------|-------------|
| libresign | features         |             |
| libresign | features_enabled |             |

If you don't have these rows in the oc_appconfig table, use this query to create the data.

```sql
insert into oc_appconfig(appid, configkey, configvalue) values('libresign', 'features', ''), ('libresign', 'features_enabled', '')
```

### CODE REVIEW

File: src/Components/Draw/Draw.vue [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+			<li
+				:class="{active: isActive('text')}"
+				@click.prevent="setActive('text')">
+				<img :src="texticon" alt="Text">
+				Text
+			</li>
+			<li
+				:class="{active: isActive('draw')}"
+				@click.prevent="setActive('draw')">
+				<img :src="drawnIcon" alt="draw">
```
Comment (vitormattos) [inline src/Components/Draw/Draw.vue:13]: I believe is a good aproach implement translation on the alt text.

File: src/Components/Draw/Draw.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="container">
+		<ul>
+			<li
+				:class="{active: isActive('text')}"
+				@click.prevent="setActive('text')">
+				<img :src="texticon" alt="Text">
```
Comment (vitormattos) [inline src/Components/Draw/Draw.vue:7]: I believe is a good aproach implement translation on the alt text.

File: src/views/Account.vue [hunk 3]
```diff
@@ -37,6 +37,47 @@
...
+
+			<div class="user sinatures">
+				<h1>{{ t('libresign', 'Your signatures') }}</h1>
+				<div class="signature-fav">
+					<header>
+						<h2>{{ t('libresign', 'Signature') }}</h2>
+						<div v-if="haveSignature" class="icon icon-rename" @click="editSignatures" />
+					</header>
+
+					<img v-if="haveSignature" :src="sinatures">
```
Comment (vitormattos) [inline src/views/Account.vue:49]: Is a good practice implement alt text focusing on accessibility

File: src/views/Account.vue [hunk 4]
```diff
@@ -37,6 +37,47 @@
...
+							{{ t('libresign', 'No signature, click here to create a new') }}
+						</span>
+					</div>
+				</div>
+				<div class="signature-fav">
+					<header>
+						<h2>{{ t('libresign', 'Initials') }}</h2>
+						<div v-if="haveInitials" class="icon icon-rename" @click="editInitials" />
+					</header>
+					<img v-if="haveInitials" :src="sinatures">
```
Comment (vitormattos) [inline src/views/Account.vue:61]: Is a good practice implement alt text focusing on accessibility

**Review by vitormattos — COMMENTED** [review]
I think it's better not to version the source binary and install it from the package:
https://fontsource.org/fonts/dancing-script

File: src/views/Account.vue (hunk 4 above)
Comment (vinicios-gomes) [reply src/views/Account.vue:61]: This PR is in Draft, it is incomplete.

File: src/views/Account.vue (hunk 3 above)
Comment (vinicios-gomes) [reply src/views/Account.vue:49]: This PR is in Draft, it is incomplete.

**Comment by vinicios-gomes** [comment]
> I think it's better not to version the source binary and install it from the package:
> https://fontsource.org/fonts/dancing-script

Very good, it's a good one. This helps with Nextcloud's Policy

File: src/Components/Draw/Draw.vue (hunk 2 above)
Comment (vinicios-gomes) [reply src/Components/Draw/Draw.vue:7]: This PR is in Draft, it is incomplete.

File: src/Components/Draw/Draw.vue (hunk 1 above)
Comment (vinicios-gomes) [reply src/Components/Draw/Draw.vue:13]: This PR is in Draft, it is incomplete.

File: src/Components/Draw/Draw.vue [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+				Text
+			</li>
+			<li
+				:class="{active: isActive('draw')}"
+				@click.prevent="setActive('draw')">
+				<img :src="drawnIcon" alt="draw">
+				Draw
```
[the remark this answers is not in the document]
Comment (vinicios-gomes) [reply src/Components/Draw/Draw.vue:14]: This PR is in Draft, it is incomplete.

File: src/Components/Draw/Draw.vue [hunk 6]
```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="container">
+		<ul>
+			<li
+				:class="{active: isActive('text')}"
+				@click.prevent="setActive('text')">
+				<img :src="texticon" alt="Text">
+				Text
```
[the remark this answers is not in the document]
Comment (vinicios-gomes) [reply src/Components/Draw/Draw.vue:8]: This PR is in Draft, it is incomplete.

**Pull request merged**